src/main/resources/application.yaml

```diff
@@ -26,13 +26,49 @@ mock:
 sts.url: ${mock.baseUrl}/sts
 veilarbabac.url: ${mock.baseUrl}/veilarbabac
 abac.url: ${mock.baseUrl}/abac
+aktørregister.url: ${mock.baseUrl}/aktoerregister/api/v1
 
-STS_BRUKERNAVN: na
-STS_PASSORD: na
+SERVICEBRUKER_BRUKERNAVN: na
+SERVICEBRUKER_PASSORD: na
+
+spring:
+  kafka:
+    consumer:
+      group-id: finn-kandidat
+
+---
+
+spring:
+  profiles: kafka-test
+  kafka:
+    bootstrap-servers: ${spring.embedded.kafka.brokers}
+
+oppfolging-avsluttet.topics:
+  - test-topic
 
 ---
 spring:
   profiles: dev
+  kafka:
+    bootstrap-servers: b27apvl00045.preprod.local:8443, b27apvl00046.preprod.local:8443, b27apvl00047.preprod.local:8443
+    client-id: finn-kandidat-api-dev
+    properties:
+      security.protocol: SASL_SSL
+      sasl:
+        mechanism: PLAIN
+        jaas.config: org.apache.kafka.common.security.plain.PlainLoginModule required username="${SERVICEBRUKER_BRUKERNAVN}" password="${SERVICEBRUKER_PASSORD}";
+      ssl.truststore:
+        location: ${javax.net.ssl.trustStore}
+        password: ${javax.net.ssl.trustStorePassword}
+    consumer:
+      group-id: finn-kandidat-api-dev
+      key-deserializer: org.apache.kafka.common.serialization.StringDeserializer
+      value-deserializer: org.apache.kafka.common.serialization.StringDeserializer
+
+oppfolging-avsluttet.topics:
```

Comment:
Her var det bare ett topic, trenger vi egentlig flere?

Reply:
veilarboppfolging publiserer i preprod til 3 topics, en for hvert testmiljø de ligger i.
Så tenkte egentlig å lytte på alle de, men er vel egentlig ikke nødvendig med flere enn feks q0...